### DISCUSSION TITLE
Unable to install the youtube revanced music

### DISCUSSION THREAD

**Discussion by sudarshanrangappa** [ReVanced/revanced-discussions#1045] [General] [answered]
![Screenshot_20240619-003532.png](https://github.com/ReVanced/revanced-discussions/assets/24605114/760afad0-3ca0-4738-a076-20e60e2b0337)

I was using this from long time and today by mistake i uninstalled revanced music but now while installing it this is showing issue.

**Comment by LisoUseInAIKyrios** [comment, marked as answer]
The manager you are using is an imposter and not official.  You might have installed malware.

Uninstall everything, change your passwords, and use the official manager from http://ReVanced.app

**Reply by sudarshanrangappa** [reply]
Ok just uninstalled it

**Reply by sudarshanrangappa** [reply]
Thanks

**Reply by sudarshanrangappa** [reply]
Installed yt music and yt . Yt music is working fine but yt is not working videos are buffering and not starting
![Uploading Screenshot_20240619-020348.png…]()

**Reply by LisoUseInAIKyrios** [reply]
If you need support check the other links on http://ReVanced.app (discord, reddit).

GitHub is for bugs and feature requests, and not intended for troubleshooting.
